nemo/utils/exp_manager.py

```diff
@@ -68,6 +68,52 @@ class CheckpointMisconfigurationError(NeMoBaseException):
     """ Raised when a mismatch between trainer.callbacks and exp_manager occurs"""
 
 
+try:
+    import dllogger
+    from dllogger import Verbosity
+
+    class DLLogger(Logger):
```

Comment:
Please define it in some utility class of nemo.utils or nemo.core.utils and then import here.

Reply:
about creating a file here: https://github.com/NVIDIA/NeMo/tree/main/nemo/utils?

Reply:
Done